README.md

```diff
@@ -1,6 +1,12 @@
-# SpeedRunMod
-hello world
+# TerrariaSpeedrunVerificationMod
 
-Hi BlobDylan!
+Features:
```

Comment:
This is a great place to start. I'm just going to leave some comments about areas I'd like expanded in the future